app/lib/frontend/handlers/headers.dart

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// TODO: move other header helpers into this library
+
+import '../../account/backend.dart';
+
+/// Represents cache-control settings for an asset category.
+class CacheHeaders {
+  /// The maximum age while a response may be cached.
+  final Duration maxAge;
+
+  /// The default cache storage category.
+  /// `public` by default
+  final String storage;
+
+  /// The cache storage category used for signed-in users.
+  /// `private by default
+  final String signedInStorage;
+
+  CacheHeaders._(
+    this.maxAge, {
+    this.storage = 'public',
+    this.signedInStorage = 'private',
```

Comment:
Since these are never used, why don't we just make private or constants, or just inline them in `call()`?